numpy/core/tests/test_multiarray.py

```diff
@@ -4380,6 +4380,12 @@ def setUp(self):
         self.omat = np.array([Decimal(repr(r)) for r in self.rmat.flat])
         self.omat = self.omat.reshape(4, 5)
 
+    def test_python_type(self):
+        for x in (np.float16(1.), 1, 1., 1+0j):
```

Comment:
> all tests are scalars wrapped into a list

Ooops, right. Great then! Good to merge from my perspective. Thank you for taking this!